core/src/main/kotlin/org/evomaster/core/output/service/TestCaseWriter.kt

```diff
@@ -97,13 +98,20 @@ abstract class TestCaseWriter {
             format.isCsharp() -> lines.add("public async Task ${test.name}() {")
         }
 
+        var anyHostnameResolutionActions = false
+
         lines.indented {
             val ind = test.test
             val insertionVars = mutableListOf<Pair<String, String>>()
-            handleFieldDeclarations(lines, baseUrlOfSut, ind, insertionVars)
+            anyHostnameResolutionActions = handleFieldDeclarations(lines, baseUrlOfSut, ind, insertionVars)
             handleActionCalls(lines, baseUrlOfSut, ind, insertionVars, testCaseName = test.name, testSuitePath)
         }
 
+        if (anyHostnameResolutionActions) {
```

Comment:
if a generated test fails, then this last statement would not be exectued, living the cache dirty. i guess should rather be in a `AfterEach` method in `TestSuiteWriter` if anywhere in the test suite there is need to use such DNS manipulation

Reply:
we are clearing at `BeforeEach`. this is unnecessary now. Will clean it,